### PR TITLE
Add json payload handling, adjust some sentences, fix lint errors

### PR DESCRIPTION
**Pull request by cziegeler**
## Description

- Added section about how clients should deal with JSON payload.
- Removed statements like "new" which are already outdated
- Linked from guide to all APIs instead just a single one
- Fix lint errors

## Motivation and Context

This is an important update for client handling of JSON

## How Has This Been Tested?

Ran all test commands provided in the readme.

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/how-to/index.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ description: How to use AEM OpenAPI-based APIs
 
 ### Introduction
 
-AEM as a Cloud Service offers a growing list of APIs that adhere to the [OpenAPI Specification](https://swagger.io/specification/v2/) (version 2), including operations on [Content Fragments](../../api/experimental/sites/) resources.
+AEM as a Cloud Service offers a variety of APIs that adhere to the [OpenAPI Specification](https://swagger.io/specification/v2/) (version 2).
```
Comment (raducotescu) [inline src/pages/guides/how-to/index.md:10]: The new APIs are OpenAPI 3.0.3. Should we rather link to https://spec.openapis.org/oas/v3.0.3?

File: src/pages/guides/how-to/index.md [hunk 2]
```diff
@@ -18,7 +20,21 @@ Credentials should be generated using the AEM Developer Console. A short-lived [
...
+
+### JSON Payloads
+
+Clients receiving JSON payload must be forgiving when processing the payload and not fail on additional properties that might not be declared in the schema the client is using. On the other hand clients must be strict when sending a JSON document as part of a request. The document must conform to the used schema.
+
+### Collections: Pagination and sorting
+
+Usually an API will support a **limit** query parameter to indicate the desired number of results to return. Documentation will state the default limit and any minimum or maximum values.
+
+If there are more resources in the collection than returend, the output result will include a parameter named **cursor**, which can be used in the LIST API call to retrieve the next set of resources in the collection.
```
Comment (raducotescu) [inline src/pages/guides/how-to/index.md:32]:
```suggestion
If there are more resources in the collection than returned, the output result will include a **cursor** property. The cursor's value can then be used in a subsequent LIST API call to retrieve the next set of resources in the collection, by passing the returned value to the **cursor** query parameter.
```

**Review by raducotescu — APPROVED** [review]

File: src/pages/guides/how-to/index.md (hunk 1 above)
Comment (cziegeler) [reply src/pages/guides/how-to/index.md]: Good point, I changed the link

**Comment by cziegeler** [comment]
@guillaumecarlino Could you please approve this change?

**Comment by guillaumecarlino** [comment]
@cziegeler , @raducotescu beat me to it :)

**Pull request merged**